.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+### Prerequisites
+
+-   [ ] Are you running the latest version?
+-   [ ] Did you perform a cursory search?
+
+For more information, see the `CONTRIBUTING` guide.
+
+### Description
+
+[Description of the bug or feature]
+
+### Impact
+
+[Description of why we should fix this bug]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:** [What you expected to happen]
+
+**Actual behavior:** [What actually happened]
+
+### Versions
+
+You can get this information from executing `npm version`.
```

Comment:
Why are we asking just for npm version here?
Node, Chrome, and Os are also required, right?